test/new_relic/http_client_test_cases.rb

```diff
@@ -629,7 +595,6 @@ def test_raw_synthetics_header_is_passed_along_when_cat_disabled
       config = {
         :encoding_key => test['settings']['agentEncodingKey'],
         :trusted_account_ids => test['settings']['trustedAccountIds'],
-        :'cross_application_tracer.enabled' => true,
         :'distributed_tracing.enabled' => false
```

Comment:
Does it make sense to turn DT on now/just return to our defaults for the synthetics tests? (everything is passing, so it's probably not too important; mostly IDK if DT needs to be off for synthetics to run)